targets/ingester/src/transform/fichesServicePublic/format.ts

```diff
@@ -1,22 +1,21 @@
 import { SOURCES } from "@socialgouv/cdtn-sources";
+import type { RawJson } from "@socialgouv/fiches-vdd-types";
+import type { IndexedAgreement } from "@socialgouv/kali-data-types";
 
-import { parseReferences } from "./parseReference.js";
+import type { ReferenceResolver } from "../../lib/referenceResolver";
+import { parseReferences } from "./parseReference";
 
-/**
- *
- * @param {import("@socialgouv/fiches-vdd-types").RawJson} element
- * @param {string} name
- */
-function getChild(element, name) {
-  return element.children.find((el) => el.name === name);
+function getChild(element: RawJson, name: string) {
+  if (element.children !== undefined) {
```

Comment:
The `children` in RawJson has been declared as non null/undefined in the index.d.ts of package `@socialgouv/fiches-vdd-types`.  So this condition is useless because a `children` is never null or undefined or the definition is wrong.

Reply:
I've updated fiches-vdd types locally but I didn't realize it was a published types so we need to the definition of RawJson